acado/nlp_solver/scp_evaluation.cpp

```diff
@@ -476,10 +499,19 @@ double SCPevaluation::getKKTtolerance(	const OCPiterate& iter,
 
     if( ( constraint != 0 ) || ( dynamicDiscretization != 0 ) )
     {
-        ((cp.lambdaConstraint.getAbsolute()).addRegularisation(eps)^cp.upperConstraintResiduum.getNegative()).getSubBlock( 0, 0, tmp, 1, 1 );
+        cp.lambdaConstraint.getAbsolute(absLC);
+		absLC.addRegularisation(eps);
```

Comment:
ugly indenting - ACADO's source files use all the indenting conventions simultaneously. Will fix.